src/components/CurrencyInput/CurrencyInputService.ts

```diff
@@ -35,15 +41,12 @@ export const createCurrencyMask = (currencyFormat, options = {}) => {
   });
 };
 
-export function formatPlaceholder(placeholder, locale, options) {
+export function formatPlaceholder(
+  placeholder?: string | number,
+  locale?: string | string[],
+  options?: Intl.NumberFormatOptions
+): string | undefined {
```

Comment:
Why not use arrow functions? 🤔  Just wondering because I just started learning TS and I'm not sure what the best practises are 😺

Reply:
It doesn't make any difference (if you ignore the `this` context). It's a personal stylistic preference — I think declaring functions with the `function` keyword makes the code more readable.

Reply:
I just remembered one difference in TypeScript: [function overloads](https://www.typescriptlang.org/docs/handbook/functions.html#overloads) only work with "traditional" `function`s, not with arrow functions.